langchain4j/src/test/java/dev/langchain4j/service/AiServicesIT.java

```diff
@@ -287,10 +288,16 @@ interface Chef {
         @UserMessage("Create recipe using only {{it}}")
         Recipe createRecipeFrom(String... ingredients);
 
+        @UserMessage(resource = "chefs-prompt-based-on-ingredients.txt")
```

Comment:
I am wondering: would it be more readable with `fromResource`: `@UserMessage(fromResource = "chefs-prompt-based-on-ingredients.txt")`? WDYT?

Reply:
I like both. Maybe yours is more in line with the style of this repo?

Reply:
I selected your suggestion.